File: blog-dao/src/main/java/com/dao/ArticleKeyMapper.java
package com.dao;

import com.entity.ArticleKey;

import java.util.List;

public interface ArticleKeyMapper {
    int deleteByPrimaryKey(Integer ktaId);

    int insert(ArticleKey record);

    int insertSelective(ArticleKey record);

    ArticleKey selectByPrimaryKey(Integer ktaId);

    int updateByPrimaryKeySelective(ArticleKey record);

    int updateByPrimaryKey(ArticleKey record);

    //通过文章id查找文章所对应的关键字id
    List<Integer> selectKeyIdByArticleId(Integer aid);

}

File: blog-dao/src/main/java/com/dao/ArticleKeyMapper.kt
package com.dao

import com.entity.ArticleKey

interface ArticleKeyMapper {
    fun deleteByPrimaryKey(ktaId: Int?): Int

    fun insert(record: ArticleKey): Int

    fun insertSelective(record: ArticleKey): Int

    fun selectByPrimaryKey(ktaId: Int?): ArticleKey

    fun updateByPrimaryKeySelective(record: ArticleKey): Int

    fun updateByPrimaryKey(record: ArticleKey): Int

    //通过文章id查找文章所对应的关键字id
    fun selectKeyIdByArticleId(aid: Int?): List<Int>

}